python/pyarrow/tests/test_orc.py

```diff
@@ -26,140 +24,13 @@
 pytestmark = pytest.mark.orc
 
 
-try:
```

Comment:
We can keep them for now but eventually these tests need to be replaced by Arrow2ORC(ORC2Arrow) ones.

Reply:
Those are slightly different, though. Roundtripping between Arrow and ORC doesn't validate that ORC data is correct, or that we are able to read foreign-produced ORC files.

Reply:
Restored.

Reply:
Well, I removed a test that compares the ORC schema with some JSON one since they don’t actually line up given the new behavior of the ORC Reader on MAP arrays.

Reply:
@pitrou Fair enough. Well, back then in Dec and Jan I did manual tests using the ORC adapter to write ORC files and then loaded them using pyorc and compared the results. We should be good.